internal/pkg/runtime/engine/oci/process_linux.go

```diff
@@ -57,6 +57,13 @@ func (e *EngineOperations) StartProcess(masterConn net.Conn) error {
 		return err
 	}
 
+	comm := os.NewFile(uintptr(masterConnFd), "master-socket")
```

Comment:
The OCI engine doesn't use the shell interpreter and globally doesn't pass large environment, so it shouldn't be less impacted by this issue, therefore I kept `net.FileConn` here to reduce the changes in this PR